deployment/sandbox-v2/helm/charts/regproc/values.template.j2

```diff
@@ -14,30 +14,96 @@ probe:
 ingress:
    namespace: default
 
-artifactoryUrl: {{ artifactory_url }} 
+artifactoryUrl: {{ artifactory_url }}
 iamAdapterUrl: {{ iam_adapter_regproc_url }}
 
-imagePullPolicy: Always 
+imagePullPolicy: Always
 
-isGlowroot: {{ is_glowroot }} 
+isGlowroot: {{ is_glowroot }}
 
 dockerHub:
-  private: {{ docker.hub.private }}  
+  private: {{ docker.hub.private }}
   keyname: {{ docker.hub.keyname }}
 
 services:
   camel:
-    serviceName: {{ versions.services.regproc.camel.keys()[0] }} 
+    serviceName: {{ versions.services.regproc.camel.keys()[0] }}
     dockerImage: {{ versions.services.regproc.camel.values()[0] }}
     ports:
       eventbusPort: 5723
       hazelcastPort: 5701
       containerPort: 8022
       servicePort: 80
-    ingressPath: '' 
+    ingressPath: ''
     healthUrl: /registrationprocessor/v1/camelbridge/actuator/health
     podconfig:
       {{ podconfig.regproc.camel | to_nice_yaml(indent=6) | indent(6) }}
+  
+  group2:
```

Comment:
These groups are already there below, you have to ensure that correct container port and health URL is configed, we cannot duplicate the groups